gomud/mob.go

```diff
@@ -5,12 +5,40 @@ import (
 	"strings"
 )
 
+/*
+	Skill defines the Name, Description, and Affect of a Mob's skill, as well
+	as the Delay from using it and the Costs of its use.
+	Name - the name of the Skill.
+	Description - text description of what the skill does.
+	Affect - how the use of the skill appears to others.
+	Delay - how many ticks the skill takes to perform.
+	Costs - how the attributes of a Mob are affected by using the skill.
+*/
 type Skill struct {
 	Name, Description, Affect string
 	Delay                     int
 	Costs                     *Attributes
 }
 
+/*
+	Mob defines the attributes of a mobile entity within the game. This can be
+	a monster, player, NPC, or anything else that moves.
+
+	ShortName - a short string naming the Mob.
+	LongName - a long string naming the Mob.
+	Room - the current Room location of the Mob.
+	Items - an array of the Items currently within the Mob's possession.
+	Equipped - the items that the Mob is currently using, represented as an
+				Equipped struct.
+	Attributes - the (maximum) Attributes of the Mob.
+	CurrentAttr - the current values of the Mob's attributes.
+	Race - the fantasy Race to which the Mob belongs.
+	Delay - the tick delay between actions for the Mob.
```

Comment:
The bug is that the Delay counter should decrease per pulse. It should be that 1-2 second delay after issuing skills or spells ('kick', 'bash', 'berserk', 'cast', etc, etc).